.github/CODE_STANDARDS.md

```diff
@@ -0,0 +1,295 @@
+- [ ] Is the code efficient, readable, and free of code smells?
+- [ ] Have all review comments been addressed in a timely manner?
+- [ ] Do you understand the code and it's purpose?
+
+This checklist serves as a guide to both authors and reviewers to ensure a thorough and effective code review process.
+
+## 3. Contributing
+
+### 3.1 Pull Requests && Issues
+
+We have separate templates for Pull Requests and Issues. Please use them when creating a new PR or Issue.
+
+### 3.2 Conventional Commits & Pull Requests Naming
+
+#### 3.2.1 Overview
+
+In an effort to maintain clarity and coherence in our commit history, we are adopting the Conventional Commits style for all commit messages across our repositories. This uniform format not only enhances the readability of our commit history but also facilitates automated tools in generating changelogs and extracting valuable information effectively.
+
+#### 3.2.2 Structure
+
+Conventional Commits follow a structured format: `type(scope): description`, where:
+
+- `type`: Represents the nature of the commit (e.g., feat, fix, chore).
+- `scope`: Denotes the relevant module or issue.
+- `description`: Provides a brief explanation of the change.
+
+When introducing breaking changes, an `!` should be appended after the `type/scope`: `feat(#123)!: introduce a breaking change`.
+
+#### 3.2.3 Types
+
+- `feat`: Utilized when introducing a new feature to the codebase.
+- `fix`: Employed when resolving a bug or issue in the code.
+- `docs`: Designated for commits involving documentation changes, such as updating README files or adding comments.
+- `style`: Applied to commits focusing on code style and formatting, without altering the code's functionality.
+- `refactor`: Used for code changes that neither introduce new features nor fix bugs, but improve the code structure or design.
+- `test`: Assigned to commits pertaining to the addition, modification, or refactoring of tests.
+- `chore`: For changes related to build processes, local development, or other maintenance tasks.
+- `perf`: Employed when enhancing the performance of the codebase.
+- `revert`: Marked for commits that revert a previous change.
+- `ci`: Applied to changes concerning the Continuous Integration (CI) configuration or scripts.
+- `deps`: Used when updating or modifying dependencies.
+
+#### 3.2.4 Scope
+
+We have standardized the use of JIRA/GitHub issue numbers as the `scope` in commits within our team. This practice aids in easily tracing the origin of changes.
+
+In the absence of an existing issue for your changes, please create one in the client’s JIRA system. If the change is not client-related, establish a GitHub issue in the repository.
+
+#### 3.2.5 Further Reading
+
+Additional information and guidelines on Conventional Commits can be found [here](https://www.conventionalcommits.org/en/v1.0.0/).
+
+#### 3.2.6 Examples
+
+##### Commit message with scope
+
+Good example:
+
+```bash
+feat: add possibility to create a new user by admin
```

Comment:
Actually there is no scope in the example

Reply:
fixed